node/pkg/aggregator/types.go

```diff
@@ -48,18 +45,7 @@ type Proof struct {
 	Proof    []byte `json:"proofs"`
 }
 
-type PgsLocalAggregate struct {
-	ConfigID  int32     `db:"configId"`
-	Value     int64     `db:"value"`
-	Timestamp time.Time `db:"timestamp"`
-}
```

Comment:
I dont see a replacement for this struct. Does that mean it was not being used anywhere else?

Reply:
My first intention was that the data format could've been different between rdb and pgsql. But later on it ended up as a same structure. You can think that this type has been merged into `types.LocalAggregate`

it was referenced in utils as followed before
```
func GetLatestLocalAggregateFromPgs(ctx context.Context, configId int32) (PgsLocalAggregate, error) {
	return db.QueryRow[PgsLocalAggregate](ctx, SelectLatestLocalAggregateQuery, map[string]any{"config_id": configId})
}
```

But now it just uses `LocalAggregate`